contracts/UFragments.sol

```diff
@@ -122,8 +125,8 @@ contract UFragments is ERC20Detailed, Ownable {
         // ever increased, it must be re-included.
         // _totalSupply = TOTAL_GONS.div(_gonsPerFragment)
```

Comment:
Let's update this commented line to assign `newTotalSupply` and move the `_totalSupply = newTotalSupply;` below it?

Reply:
done 👍